shared/src/chain_state.rs

```diff
@@ -103,6 +103,10 @@ impl<CS: ChainStore> ChainState<CS> {
         })
     }
 
+    pub fn get_store(&self) -> Arc<CS> {
```

Comment:
This breaks the intention that we must explicitly clone a store.

Reply:
Keep the interface consistent with `Shared::store`

Reply:
Renamed `get_store()` -> `store()`